server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinator.java

```diff
@@ -538,10 +538,9 @@ List<CoordinatorDuty> makeIndexingServiceDuties()
     if (getCompactSegmentsDutyFromCustomGroups().isEmpty()) {
       duties.add(compactSegments);
     }
-    log.debug(
-        "Initialized indexing service duties [%s].",
-        duties.stream().map(duty -> duty.getClass().getName()).collect(Collectors.toList())
-    );
+    if (log.isDebugEnabled()) {
+      log.debug("Initialized indexing service duties [%s].", duties.stream().map(duty -> duty.getClass().getName()).collect(Collectors.toList()));
```

Comment:
This one was better broken up across multiple lines. It seems difficult to read now.